x-pack/plugins/security_solution/public/alerts/components/rules/step_define_rule/translations.tsx

```diff
@@ -23,7 +23,7 @@ export const INVALID_CUSTOM_QUERY = i18n.translate(
 export const CONFIG_INDICES = i18n.translate(
   'xpack.securitySolution.detectionEngine.createRule.stepDefineRule.indicesFromConfigDescription',
   {
-    defaultMessage: 'Use Elasticsearch indices from SIEM advanced settings',
+    defaultMessage: 'Use Elasticsearch indices from Security advanced settings',
```

Comment:
I wonder if we want to be explicit and say `Security Solution` here. @benskelker, how will you be referring to advanced settings now in the docs, will you be using `Security advanced settings` or `Security Solution advanced settings`? 
<p align="center">
  <img width="500" src="https://user-images.githubusercontent.com/2946766/84084072-090e1200-a9a0-11ea-8e51-fb55f0c5176a.png" />
</p>

Reply:
@spong - sorry missed this. We'll go with what's in the UI, so `Security Solution advanced settings`.